File: src/1480-Running-Sum-of-1d-Array/running_sum_of_1d_array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
  var output = []
  for (let i = 0; i < nums.length; i++) {
      if (i == 0) {
        output.push(nums[i])
      }
      else {
        output.push(nums[i] + output[i-1])
      }
  }
return output
}

nums = [1,2,3,4]
console.log(runningSum(nums))

File: src/1480-Running-Sum-of-1d-Array/running_sum_of_1d_array.py
from typing import List

class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        output = []
        for i in range(len(nums)):
            if i == 0:
                output.append(nums[i])
            else:
                output.append(nums[i] + output[i-1])

        return output

if __name__ == "__main__":
    nums: List[int] = [1,2,3,4]
    solution = Solution()
    print(solution.runningSum(nums))